src/panels/profile/ha-pick-number-format-row.ts

```diff
@@ -45,15 +45,15 @@ class NumberFormatRow extends LitElement {
           >
             ${Object.values(NumberFormat).map((format) => {
               const formattedNumber = formatNumber(1234567.89, {
-                language: this.hass.locale.language,
+                ...this.hass.locale,
                 number_format: format,
               });
               const value = this.hass.localize(
                 `ui.panel.profile.number_format.formats.${format}`
               );
               const twoLine = value.slice(value.length - 2) !== "89"; // Display explicit number formats on one line
               return html`
-                <paper-item .format=${format}>
+                <paper-item .format=${format} .label=${value}>
```

Comment:
Without it, the label was both the format name and the example value combined => once you selected a short format such as "None", the selected dropdown item actually also showed the formatted number example as well. Now only the format name itself is the label.